.gitignore

```diff
@@ -1,4 +1,3 @@
 .vscode/
 node_modules/
-dist
```

Comment:
Need to remove this so that the post install build directory (dist) is not deleted.